src/Cart-update/reducers/date.reducer.js

```diff
@@ -0,0 +1,39 @@
+import {
+    ADD_QUANTITY,
+    REDUCE_QUANTITY,
+    DELETE_ITEM
+
+} from '../constants';
+
+const initialState = {
+    items: [
+        {name: 'item1', qty: 2, price: 30},
+        {name: 'item2', qty: 5, price: 10},
+        {name: 'item3', qty: 7, price: 120}
+      ]
+};
+
+const dateReducer = function (state = initialState, action) {
+  switch (action.type) {
+    case ADD_QUANTITY:{
+    	let item_list = state.items;
+    	item_list[action.add].qty++;
+    	return { ...state, item_list };
```

Comment:
@nerdchacha  Thanks for providing the feedback, I have updated the code as per provided feedback.
Please check and confirm